fs/span-manager/span_manager.go

```diff
@@ -98,6 +100,7 @@ type SpanManager struct {
 	r        *io.SectionReader // reader for contents of the spans managed by SpanManager
 	spans    []*span
 	ztoc     *ztoc.Ztoc
+	retries  int
```

Comment:
Could we get a comment here? Is this max retries? Number of retries so far? Something else? I can figure it out from the code, but that's a lot of effort to figure out the meaning of the struct.